test/common/markdown/package-lock.json

```diff
@@ -25,6 +25,15 @@
       "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-2.1.1.tgz",
       "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8="
     },
+    "ansi-styles": {
```

Comment:
Do we really need the file package-lock.json?

Reply:
Yes, it speeds up installing

Reply:
Also dependabot an only do 1 language

Reply:
dependabot does actually support multiple languages.  I added this folder and we will see the results.

Reply:
Got it.  If it tries to bump the folder I will approve it's PRs.

Reply:
Also, our compliance team wants them